test/sync/src/commonMain/kotlin/io/realm/mongodb/sync/SyncDispatcher.kt

```diff
@@ -0,0 +1,5 @@
+package io.realm.mongodb.sync
+
+import kotlinx.coroutines.CoroutineDispatcher
+
+expect fun getDispatcher(): CoroutineDispatcher
```

Comment:
You could just use `io.realm.internal.platform.singleThreadDispatcher` from the library?

Reply:
Yes, that's what I'm doing in the next PR.